src/chatgpt-api.ts

```diff
@@ -286,7 +287,17 @@ export class ChatGPTAPI {
           }
         }
       }
-    ).then((message) => {
+    ).then(async (message) => {
+      if (message.detail && !message.detail.usage) {
+        const promptTokens = numTokens
+        const completionTokens = await this._getTokenCount(message.text)
+        message.detail.usage = {
+          prompt_tokens: promptTokens,
+          completion_tokens: completionTokens,
+          total_tokens: promptTokens + completionTokens,
+          estimated: true
```

Comment:
Mark these values as `estimated` to let user know that they did not come from OpenAI, so it means the only "theoretically" represent usage data in OpenAI account

Reply:
makes sense